worker/httpserver/manifold.go

```diff
@@ -99,11 +111,27 @@ func (config ManifoldConfig) start(context dependency.Context) (_ worker.Worker,
 		return nil, errors.Trace(err)
 	}
 
-	var clock clock.Clock
-	if err := context.Get(config.ClockName, &clock); err != nil {
+	var mux *apiserverhttp.Mux
+	if err := context.Get(config.MuxName, &mux); err != nil {
 		return nil, errors.Trace(err)
 	}
 
+	// We don't actually need anything from these workers, but we
+	// shouldn't start until they're available.
+	if err := context.Get(config.APIServerName, nil); err != nil {
```

Comment:
I'm kinda surprised that this works, but I'm pleased it does.

Reply:
Yeah, same - it's documented in the interface, so the behaviour's deliberate. Thanks Will!